src/Viewer/services/decoder/FileManager.js

```diff
@@ -156,8 +156,10 @@ class FileManager {
         }
         if (logEventIdx > this._state.numberOfEvents) {
             console.debug("Log event provided was larger than the number of events.");
-            console.debug(`Log event was limited to ${this._state.numberOfEvents}`);
-            this._state.logEventIdx = this._state.numberOfEvents;
+            this._updateStateCallback(CLP_WORKER_PROTOCOL.UPDATE_STATE, this._state);
```

Comment:
Why do we need this call and the one on 162 since we're calling it regardless on line 174?

Reply:
Because it wouldn't have to execute the code between 162 and 174

Reply:
but I guess it doesn't matter, its not a heavy operation

Reply:
Done.

Reply:
Uh, is your intention to have a return statement in the if blocks then?

Reply:
what do you mean? I removed them in my commit

![image](https://user-images.githubusercontent.com/101469936/222409770-dca9f883-a650-4335-81ca-80d090474d5b.png)

Reply:
Oh sorry, I just understood what you mean, yea I did intend to have them in the if statement but writing it this way is fine as well.